src/restLevelClient/generateClientDefinition.ts

```diff
@@ -247,12 +250,12 @@ function getOperationOptionsType(
 
 function getOperationReturnType(
   operation: Operation,
-  importedResponses = new Set<string>()
+  importedResponses = new Set<string>(),
+  coreClientImports = new Set<string>()
 ) {
   let returnType: string = "HttpResponse";
-  if (operation.responses && operation.responses.length) {
-    const responses = [...operation.responses, ...(operation.exceptions || [])];
-
+  if ((operation.responses && operation.responses.length) || (operation.exceptions && operation.exceptions.length)) {
+    const responses = [...(operation.responses || []), ...(operation.exceptions || [])];
```

Comment:
nit: ` [...(operation.responses ?? []), ...(operation.exceptions ?? [])];` makes the intent clearer.

Reply:
updated with prettier.